test/lib/ActionGenerator.test.js

```diff
@@ -17,6 +17,21 @@ jest.mock('yeoman-generator')
 
 const ActionGenerator = require('../../lib/ActionGenerator')
 const Generator = require('yeoman-generator')
+const mockfs = {
```

Comment:
all those mocks should be restored as part of the beforeEach in promptForActionName IMO